host_cleanup.sh

```diff
@@ -31,40 +31,37 @@ if [ "${CAPM3_RUN_LOCAL}" = true ]; then
   fi
 fi
 
-
 ANSIBLE_FORCE_COLOR=true ansible-playbook \
-    -e "working_dir=$WORKING_DIR" \
-    -e "num_nodes=$NUM_NODES" \
-    -e "extradisks=$VM_EXTRADISKS" \
-    -e "virthost=$HOSTNAME" \
-    -e "manage_baremetal=$MANAGE_BR_BRIDGE" \
-    -e "nodes_file=$NODES_FILE" \
-    -i vm-setup/inventory.ini \
-    -b -v vm-setup/teardown-playbook.yml
+  -e "working_dir=$WORKING_DIR" \
+  -e "num_nodes=$NUM_NODES" \
+  -e "extradisks=$VM_EXTRADISKS" \
+  -e "virthost=$HOSTNAME" \
+  -e "manage_baremetal=$MANAGE_BR_BRIDGE" \
+  -e "nodes_file=$NODES_FILE" \
+  -i vm-setup/inventory.ini \
+  -b -v vm-setup/teardown-playbook.yml
 
-if [ "$USE_FIREWALLD" == "False" ]; then
- ANSIBLE_FORCE_COLOR=true ansible-playbook \
-    -e "{use_firewalld: $USE_FIREWALLD}" \
-    -e "external_subnet_v4: ${EXTERNAL_SUBNET_V4}" \
-    -e "firewall_rule_state=absent" \
-    -i vm-setup/inventory.ini \
-    -b -v vm-setup/firewall.yml
-fi
+ANSIBLE_FORCE_COLOR=true ansible-playbook \
+  -e "{use_firewalld: $USE_FIREWALLD}" \
```

Comment:
Remove use of JSON syntax, liike in Adam's patch? It is bad habit to mix syntaxes as we see on the next line.